Mathlib/Analysis/Meromorphic/Basic.lean

```diff
@@ -597,4 +598,35 @@ theorem eventually_codiscreteWithin_analyticAt
   intro x hx
   simp [hx]
 
+/--
+The singular set of a meromorphic function is countable.
+-/
+theorem countable_compl_analyticAt [SecondCountableTopology 𝕜] [CompleteSpace E]
+    (h : MeromorphicOn f U) :
+    ({z | AnalyticAt 𝕜 f z}ᶜ ∩ U).Countable := by
+  classical
+  have : DiscreteTopology ↑({z | AnalyticAt 𝕜 f z}ᶜ ∩ U) := by
+    apply isDiscrete_iff_discreteTopology.1 (isDiscrete_of_codiscreteWithin _)
+    simpa using eventually_codiscreteWithin_analyticAt f h
+  apply countable_of_Lindelof_of_discrete
```

Comment:
Can you please provide a `IsDiscrete` + `Set.Countable` version of `countable_of_Lindelof_of_discrete` (which is itself misnamed because the `L` should be lower case). That will make this much cleaner.